packages/admin/src/routes/entities/components/DetailView/DetailView.js

```diff
@@ -12,8 +13,13 @@ const DetailViewContainer = styled.div`
 `
 
 const DetailViewPart = styled.div`
-  margin: 5px;
   flex: 1;
+  margin-right: 1rem;
+  
+  .tocco-entity-detail {
```

Comment:
we usually do not use class selectors when working with StyledComponents. Is there a particular reason why?